### PR TITLE
dehardcoded server domain name

### PR DESCRIPTION
**Pull request by akhon**

### CODE REVIEW

File: templates/verygoodproxy.conf.j2 [hunk 1]
```diff
@@ -16,27 +16,27 @@ server {
 
 server {
     listen              80;
-    server_name         ~^(?<tenant>.+)\.(?<data_environment>(live|sandbox))\.verygoodproxy\.(?<tld>(io|com))$;
+    server_name         ~^(?<tenant>.+)\.(?<data_environment>(live|sandbox))\.(?<domain_name>.+)\.(?<tld>(io|com))$;
```
Comment (zdmytriv) [inline templates/verygoodproxy.conf.j2]: we need to make it work for domains like this `tntf30c7368.proxy.live.vgs.lendupvault.com`

```
card-statements.lendupvault.com --> CNAME --> tntf30c7368.proxy.live.vgs.lendupvault.com --> REVERSE PROXY --> card-statements-api.lendup.com
```

Comment (akhon) [reply templates/verygoodproxy.conf.j2]: `.vgs.`
ugh. ok.

Comment (akhon) [reply templates/verygoodproxy.conf.j2]: @zdmytriv check it out, line #43

Comment (zdmytriv) [reply templates/verygoodproxy.conf.j2]: screen shot 2018-08-20 at 1 02 57 pm

Line 43 ^, what is wrong there?

File: templates/verygoodproxy.conf.j2 [hunk 2]
```diff
@@ -16,27 +16,55 @@ server {
...
 
-    set $tenant_endpoint $tenant.$data_environment.verygoodproxy.$tld;
+    set $tenant_endpoint $tenant.$data_environment.$domain_name.$tld;
+    access_log /var/log/nginx/access.log json_combined;
+}
+
+server {
+    listen              80;
+    # tntf30c7368.proxy.live.vgs.lendupvault.com
+    server_name         ~^(?<tenant>.+)\.proxy\.(?<data_environment>(live|sandbox))\.vgs\.(?<domain_name>.+)\.(?<tld>(com))$;
```
Comment (zdmytriv) [inline templates/verygoodproxy.conf.j2]: I would hardcode Lendup here. This wouldn't be common case to create domains like this.
```
~^(?<tenant>.+)\.proxy\.(?<data_environment>(live|sandbox))\.vgs\.(?<domain_name>.+)\.(?<tld>(com))$;
```
replace with
```
~^(?<tenant>.+)\.proxy\.(?<data_environment>(live|sandbox))\.vgs\.lendupvault\.com)$
```
it was mistake that we made and now we have to deal with it.

Comment (akhon) [reply templates/verygoodproxy.conf.j2]: i would keep this patter for future use.

**Pull request merged**